kcp/controllers/assets/istio.networking.crds.yaml

```diff
@@ -0,0 +1,1852 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
As you mentioned please remove this..

Reply:
I added this CRDs file for the tests in `watcher_controller_test.go` to run. not for local testing.

Reply:
Please remove this IMO. You could make an http call to the crd source to get the latest resource from istio.